aux/vagrant-build/Vagrantfile

```diff
@@ -1,67 +1,32 @@
 # vim: sw=2:ts=2:et:ft=ruby
 
-boxes = [
-  {:name => 'precise',  :libvirt => 'fm-ubuntu1204', :image_name => /Ubuntu.*12\.04/, :os_user => 'ubuntu'},
-  {:name => 'squeeze',  :libvirt => 'fm-debian6',    :image_name => /Debian.*6/,      :os_user => 'debian'},
-  {:name => 'wheezy',   :libvirt => 'fm-debian7',    :image_name => /Debian.*7/,      :os_user => 'debian'},
-  {:name => 'f27',      :libvirt => 'fm-fedora25',   :image_name => /Fedora.*27.*PVHVM/, :pty => true},
-  {:name => 'f28',      :libvirt => 'fm-fedora25',   :image_name => /Fedora.*28.*PVHVM/, :pty => true},
-  {:name => 'el6',      :libvirt => 'fm-centos64',   :image_name => /CentOS 6\.5 SELinux/, :default => true, :pty => true},
-  # There is no livecd-tools available in EL 7.0-7.3 - use Fedora instead
-  {:name => 'el7',      :libvirt => 'fm-centos70',   :image_name => /CentOS 7/, :default => true, :pty => true},
-]
-
-if ENV['box']
-  boxes << {:name => ENV['box'], :libvirt => ENV['box'], :image_name => ENV['box'], :os_user => ENV['box']}
-end
-
 SHELLARGS = []
 SHELLARGS << ENV['repoowner'] || ''
```

Comment:
Note you want to use `<< (ENV['repoowner'] || '')` because otherwise you still end up with an array of `[nil, nil, nil]`